extensions/amp-form/0.1/amp-form.js

```diff
@@ -260,17 +316,114 @@ function onInvalidInputBlur_(event) {
  * @param {!HTMLInputElement} input
  */
 function reportInputValidity(input) {
-  input./*OK*/focus();
+  if (isReportValiditySupported()) {
+    input.reportValidity();
+  } else {
+    input./*OK*/focus();
+
+    // Remove any previous listeners on the same input. This avoids adding many
+    // listeners on the same element when the user submit pressing Enter or any
+    // other method to submit the form without the element losing focus.
+    input.removeEventListener('blur', onInvalidInputBlur_);
```

Comment:
Yes that's possible. If we do that we can drop the remove event listener stuff. This relates to previous PR so I'd rather make the change for this in next PRs. Filed an issue to track this #4305